### PR TITLE
Rearrange changelog

### PR DESCRIPTION
**Pull request by AbhijitNage123**

### CODE REVIEW

**Review by RajkiranBagal — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -173,16 +173,19 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
...
 * Improvement: Countdown - Added the Countdown block to the list of Core Blocks.
 * Improvement: Typography - Added 'Bebas Neue' Google font in Font Families.
+* Fix: Container - Content box width does not apply in the editor inside column core block.
 * Fix: Container - Resolved an issue where the video/image background styling was not working as expected on the front-end.
 * Fix: Image - Resolved an issue where an error would occur when we try to crop the image.
 * Fix: Buttons - Fixed the issue in which the text hover color would apply to the icon when the icon is enabled.
 * Fix: Info Box -  Resolved an issue in which the prefix margin would not be applied on the front-end.
 * Fix: Forms - Fixed an issue on form submission failure in FSE themes.
 * Fix: Image Gallery - Resolved an issue wherein the carousel images would initially appear big on load.
+* Fix: Masonry Gallery - Resolved an issue in which the Masonry Gallery extension would not work as intended in the FSE.
```
Comment (bsf-zanev) [inline README.md:183]: Should Masonry Gallery be here?
Also should we have it like this?

```suggestion
* Fix: Masonry Gallery Extension - Resolved an issue in which the Masonry Gallery extension would not work as intended in the FSE.
```
It's an extension, not a block.

Comment (AbhijitNage123) [reply README.md]: Resolved

**Review by bsf-zanev — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -173,16 +173,19 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 * Improvement: Heading - Added the functionality to move onto a new paragraph when pressing the enter key inside the Heading block.
 * Improvement: Countdown - Added the Countdown block to the list of Core Blocks.
 * Improvement: Typography - Added 'Bebas Neue' Google font in Font Families.
+* Fix: Container - Content box width does not apply in the editor inside column core block.
```
Comment (bsf-zanev) [inline README.md:176]:
```suggestion
* Fix: Container - Resolved an issue in which the content box width would not apply in the editor inside the column core block.
```

Comment (AbhijitNage123) [reply README.md]: Resolved

**Review by bsf-zanev — APPROVED** [review]

**Pull request merged**